src/index.js

```diff
@@ -74,19 +86,96 @@ function doThis() {
   $('.total_rooms').text(hotel.totalRoomsAvailableToday(currentDate));
   $('.total_rev').text(hotel.totalRevenueToday(currentDate));
   $('.total_perc').text(hotel.percentRoomsOccupied(currentDate));
-  console.log(hotel.users)
 }
 
 function orThis() {
   $('.login_form').toggle('hidden');
   $('header').toggle('hidden');
   $('.customer_dashboard').toggle('hidden');
-  let id = parseInt($('.login_username').val())
-  hotel.findUser(id)
-  user = new User(hotel.currentUser, hotel.bookings, hotel.rooms)
-  console.log(user)
-  $('.header_p_span').text(user.name.split(' ')[0])
-  $('.customer_total').text(user.totalSpent())
+  let id = parseInt($('.login_username').val());
+  hotel.findUser(id);
+  user = new User(hotel.currentUser, hotel.bookings, hotel.rooms);
+  $('.header_p_span').text(user.name.split(' ')[0]);
+  $('.customer_total').text(user.totalSpent());
+  displayBookings()
 }
 
+function displayBookings() {
+  let allBookings = user.allUserBookings()  
+  allBookings.forEach(currentBooking => {
+      $('.booking_body').append(`
+      <tr class="room-number" data-room="${currentBooking.roomNumber}">
+      <td>${currentBooking.roomNumber}</td>
+      <td>${currentBooking.date}</td>
+      </tr>
+      `)
+    })
+  }
+
+function sortByType(table, prop) {
+  let rooms = table
+  if(prop === 'roomType') {
+    return rooms.sort((a, b) => {
```

Comment:
I'm so used to [a - b] or [a + b} with the sort() method that I didn't even know that you could use operators like <. Learned something useful today